### DISCUSSION TITLE
[v2] Why port 3000 as default?

### DISCUSSION THREAD

**Discussion by italodeandra** [coollabsio/coolify#128] [Feature Requests]
Did I do anything wrong? Do I need to manually change the docker's port to 80 to be able to use a domain directly for my coolify instance?

**Comment by italodeandra** [comment]
I'm dumb. The proxy is automatically created on port 80.

**Reply by andrasbacsai** [reply]
You are not dumb. Coolify lacks of documentation. 😃

**Reply by jub0t** [reply]
Is/will there (be) a way to change the default port?

**Reply by andrasbacsai** [reply]
Yes, [it will be](https://github.com/coollabsio/coolify/issues/382).

**Comment by peaklabs-dev** [comment]
@italodeandra Changing the default port from 8000 (v4) is possible with the following environment variable: `APP_PORT=`
